docs/deployment/kubernetes.md

```diff
@@ -30,6 +30,92 @@ We have a special guide for minikube here:
       --user="$(gcloud config get-value core/account)"
     ```
 
+#### How to use a private registry with Kubernetes
+
+If you are using a hosted private Docker registry ([Docker Hub](https://hub.docker.com/), or other),
+in order to check how to cofigure it, please visit the Kubernetes [documentation](https://kubernetes.io/docs/concepts/containers/images/#using-a-private-registry).
+
+#####  Deploy a function from a private Docker image
+
+With the following commands you can deploy a function from a private Docker image, tag and push it to your docker registry account:
+
+```bash
+$ docker pull functions/alpine:latest
+$ docker tag functions/alpine:latest $DOCKER_USERNAME/private-alpine:latest
+$ docker push $DOCKER_USERNAME/private-alpine:latest
+```
+
+Log into the [Hub](https://hub.docker.com/) and make your image `private-alpine` private.
+
+Then create your openfaas project:
+
+```bash
+$ mkdir privatefuncs && cd privatefuncs
+$ touch stack.yaml
+```
+
+In your favoriate editor, open stack.yaml and add
```

Comment:
Please could you run a spell-check on your work in future PRs? `favoriate` is a typo.